client/src/Atoms/Rocket/RocketStyled.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const RocketStyled = styled.div`
+  transform: translate(0%);
+  transition: 0.3s ease-out;
+
+  &:hover {
+    transform: translate(-5%, -30%);
+    transition: 0.3s ease-out;
```

Comment:
You should add transform property to transition. We shouldn't animate all things :)